### DISCUSSION TITLE
"export 'decorate' was not found in 'mobx'

### DISCUSSION THREAD

**Discussion by yoni-wibbitz** [mobxjs/mobx#2623] [Q&A]
I am trying to upgrade from mobx 5 to mobx 6 and after the upgrade my build isn't working as expected. I am getting  this warning:
"WARNING in ../node_modules/mobx-cookie/dist-web/index.js 106:0-8
"export 'decorate' was not found in 'mobx'
 @ multi @blueprintjs/core @loadable/component @researchgate/react-intersection-observer @stripe/react-stripe-js @stripe/stripe-js auth0-lock autosize bricks.js case-sensitive-paths-webpack-plugin classnames coloreact cropperjs deep-object-diff dropzone element-resize-detector grunt-exec gsap i18next i18next-browser-languagedetector jest-transform-stub jquery jssha justified-layout launchdarkly-js-client-sdk libphonenumber-js lodash lottie-web measure-font mobx mobx-cookie mobx-react moment prop-types pusher-js query-string react react-autosuggest react-beautiful-dnd react-day-picker react-dnd react-dnd-html5-backend react-dnd-touch-backend react-dom react-flip-move react-i18next react-router react-router-dom react-slick react-transition-group react-virtualized reactour regenerator-runtime sanitize-html smooth-scrollbar styled-components tinycolor2 tti-polyfill web-vitals
"

does the upgrade suppose to impact all of those packages?

**Comment by mweststrate** [comment]
Sounds like something that should be picked up with mobx-cookie? decorate
doesn't exist anymore indeed in MobX 6.
